13/umbraco-deploy/deploy-settings.md

```diff
@@ -292,13 +279,28 @@ Combinations of settings can be applied, e.g. `Hostname,AbsolutePath`.
 Webhooks may be considered environment specific or schema information that you would like to synchronize between environments. As such, by default, Umbraco Deploy does not include webhooks in schema deployment operations.
 
 ```json
-    "AllowWebhooksDeploymentOperations": "None|All",
+"AllowWebhooksDeploymentOperations": "None|All",
 ```
 
 If you would like you include them you can adjust this setting:
 
-* _None_ - webhooks are not deployed and are expected to be managed independently in each environment.
-* _All_ - webhooks included in schema deployments.
+* `None` - webhooks are not deployed and are expected to be managed independently in each environment.
+* `All` - webhooks included in schema deployments.
+
+## Deployment of trashed content
+
+Specifies options for handling trashed content (documents, media and members) on export or import:
+
+```json
+"TrashedContentDeploymentOperations": "None|Export|Import|All"
+```
+
+You can amend this behavior using this setting:
+
+* `None` - trashed content will not be exported or imported
+* `Export` - trashed content will be included in an export
+* `Import` - trashed content will be processed and moved to the recycle bin on import
+* `All` - trashed content will be included in an export, processed and moved to the recycle bin on import
```

Comment:
this change has been added in PR https://github.com/umbraco/UmbracoDocs/pull/5750